cuda_core/tests/test_memory.py

```diff
@@ -437,3 +441,11 @@ def test_mempool_attributes_ownership(mempool_device):
     with pytest.raises(RuntimeError, match="DeviceMemoryResource is expired"):
         _ = attributes.used_mem_high
     mr._mempool_handle = old_handle
+
+
+# Ensure that memory views dellocate their reference to dlpack/cupy tensors
+@pytest.mark.skipif(cupy is None, reason="cupy is not installed")
+def test_strided_memory_view_leak(benchmark):
```

Comment:
Is the `benchmark` fixture intentional here?

Reply:
Nope.  Removing.